src/v2/views/RegisterView/index.tsx

```diff
@@ -64,6 +64,7 @@ function RegisterView() {
         address={key?.address}
         onSubmit={onSubmit}
         useActivitionKey
+        useLink={"https://nine-chronicles.com/start?step=2"}
```

Comment:
```suggestion
        urlForActivationCode={"https://nine-chronicles.com/start?step=2"}
```
it seems that `use`-prefix  has been intended for `bool` (e.g., `useActivation`)

Reply:
I just removed this specific option entirely from this component because it's usage seems too narrow. instead I moved it to RegisterView itself.